server/main.go

```diff
@@ -42,10 +44,16 @@ func main() {
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
-	rpcEndpoint := fmt.Sprintf("%s:%d", *host, *rpcPort)
+	conn, err := grpc.Dial(*mlmdAddr, grpc.WithInsecure())
+	if err != nil {
+		glog.Fatalf("fail to dial to MLMD server: %v", err)
```

Comment:
Nit: "Failed to connect to MLMD server:...."

Reply:
Done